### PR TITLE
Delete pip and onboarding updates

### PR DESCRIPTION
**Pull request by gcnabeel**

### CODE REVIEW

File: Datahub.Core/Model/Onboarding/Onboarding_Constants.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    public class Onboarding_Constants
+    {
+        public static readonly string[] CATEGORY = { "Data Pipeline",
+                                                    "Data Science",
+                                                    "Full Stack",
+                                                    "Guidance",
+                                                    "Power BI Reports",
+                                                    "Storage",
+                                                    "Web Forms",
+                                                    "UNknown",
```
Comment (gcalexk) [inline Datahub.Core/Model/Onboarding/Onboarding_Constants.cs:18]: The capitalization should be "Unknown"

File: Datahub.Core/Model/Onboarding/Onboarding_Constants.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                                                    "Other"
+        };
+
+        public static readonly string[] SECURITYLEVEL = {   "Classified", 
+                                                            "Protected A",
+                                                            "Protected B",
+                                                            "Protected C",
+                                                            "UNclassified",};
```
Comment (gcalexk) [inline Datahub.Core/Model/Onboarding/Onboarding_Constants.cs:26]: Same with this one

File: Datahub.Core/Model/Onboarding/Onboarding_Constants.cs (hunk 1 above)
Comment (gcnabeel) [reply Datahub.Core/Model/Onboarding/Onboarding_Constants.cs:18]: Thanks!

File: Datahub.Core/Model/Onboarding/Onboarding_Constants.cs (hunk 2 above)
Comment (gcnabeel) [reply Datahub.Core/Model/Onboarding/Onboarding_Constants.cs:26]: lol.. thanks :)

**Review by hperezga — APPROVED** [review]

**Review by gcalexk — APPROVED** [review]

**Pull request merged**